.travis.yml

```diff
@@ -12,6 +12,18 @@ env:
   - OCAML_VERSION=4.05
   - OCAML_VERSION=4.06
   - OCAML_VERSION=4.07
+matrix:
+  exclude:
```

Comment:
Given the builds we want to run I think the include syntax would be a better a fit as it would be more concise. I'm not deeply familiar with the fine grained tweaks of travis config but would the following work?
```yaml
env:
  global:
  - PACKAGE=yojson

matrix:
  include:
  - os: linux
    env: OCAML_VERSION=4.02 OCAML_SWITCH=4.02.3
  - os: linux
    env: OCAML_VERSION=4.03
  - os: linux
    env: OCAML_VERSION=4.04
  - os: linux
    env: OCAML_VERSION=4.05
  - os: linux
    env: OCAML_VERSION=4.06
  - os: linux
    env: OCAML_VERSION=4.07
  - os: osx
    env: OCAML_VERSION=4.07
```

Couldn't find the answer in their documentation unfortunately. Would you mind trying that?